README.md

```diff
@@ -1,170 +1,508 @@
 
-![image](https://user-images.githubusercontent.com/16775804/36432698-b50c54f6-1652-11e8-8330-513c06150d05.png)
+Scroll downd to the "About" section you will find the `ID` of your app e.g: 103
+this is the number that needs to be set for `GITHUB_APP_ID` in your `.env` file.
 
+![app-about-id](https://user-images.githubusercontent.com/194400/37863366-004a4f02-2f55-11e8-97eb-7d9f8b74ba66.png)
 
-- Select which events ```dwylbot``` will be notified on. **Check "issues" and "issue comment"**
 
-  ![image](https://user-images.githubusercontent.com/16775804/36432733-d4901592-1652-11e8-841f-06e7b4bf9b4c.png)
 
-- Webhook secret: This token can be used to make sure that the requests received by `github-backup` are from Github; `github-backup` doesn't use this token at the moment so you can keep this field empty (see https://developer.github.com/webhooks/securing/)
 
-- You can decide to allow other users to install the Github Application or to limit the app on your account only:
-  ![Github app install scope](https://user-images.githubusercontent.com/6057298/34677046-cf874e96-f486-11e7-9f60-912f3ec2809b.png)
 
-  You can now click on "Create Github App"!
+#### Generate the `SECRET_KEY_BASE`
 
-- Create a private key: This key is used to identify specific `github-backup` installations
+Run the following command to generate a new phoenix secret key:
 
-  ![Github App private key](https://user-images.githubusercontent.com/6057298/34678365-d9d73dd0-f48a-11e7-8d1b-cfbfa11bbcc9.png)
+```sh
+mix phx.gen.secret
+```
+_copy-paste_ the _output_ (64bit `String`)
+into your `.env` file after the "equals sign" on the line for `SECRET_KEY_BASE`:
+```yml
+export SECRET_KEY_BASE=YourSecreteKeyBaseGeneratedUsing-mix_phx.gen.secret
+```
 
-  The downloaded file contains the private key.
-  Copy this key in your environment variables, see the next section.
+####  S3 Bucket
 
+In order to save the Issue Comments (_which are a variable length "rich text"_),
+we save these as `.json` in an S3 bucket. If you do not _already_ have
+an S3 bucket where you can store issues/comments, create one now.
 
-You can also read the Github guide on how to create a new Github App at https://developer.github.com/apps/building-github-apps/creating-a-github-app/
+Once you have an S3 bucket, copy the name and paste it into your `.env` file
+against the `S3_BUCKET_NAME` environment variable.
 
-#### Run a `github-backup` server
+You will _also_ need to define the
 
-The `github-backup` server will receive events from Github and then save the details or edits to an external database.
+#### "Source" the `.env` File
 
-- Clone the repository _to your personal computer_:
-  ```
-  git clone git@github.com:dwyl/github-backup.git && cd github-backup
-  ```
-- Define the local environment variables:
+Now that you have defined all the necessary
+environment variables in your `.env` file, <br /
```

Comment:
This `<br /` is not closed, either close it or remove it 🤔

Reply:
fixed.